### PR TITLE
added pr template

### PR DESCRIPTION
**Pull request by Sloth-Panda**
corresponds to issue -> #1 

@luckykumarirai

### CODE REVIEW

**Review by luckykumarirai — CHANGES_REQUESTED** [review]
Remove these parts it makes too lengthy, make it simple.
add 2 or 3 checklists only

> which parts @luckykumarirai


- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

**Comment by Sloth-Panda** [comment]
which parts @luckykumarirai

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
+A clear and concise description of it.
+
+## Checklist:
+<!--
+Example how to mark a checkbox:-
+- [x] My code follows the code style of this project.
+-->
+- [ ] My code follows the style guidelines of this project.
```
Comment (luckykumarirai) [inline .github/pull_request_template.md:33]: Add this checklist.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- [ ] I have performed a self-review of my own code.
+- [ ] I have commented my code, particularly in hard-to-understand areas.
+- [ ] I have made corresponding changes to the documentation.
```
Comment (luckykumarirai) [inline .github/pull_request_template.md:36]: Add these 3 also.

**Comment by luckykumarirai** [comment]
Can you work on this ASAP @Sloth-Panda? I need a proper template to make it easy for me to review PRs.

**Comment by Sloth-Panda** [comment]
@luckykumarirai check it now

**Comment by luckykumarirai** [comment]
Nice work!

**Pull request merged**